packages/web-react/src/components/Dropdown/README.md

```diff
@@ -9,23 +10,23 @@ import { Dropdown } from '@lmc-eu/spirit-web-react/components';
-| ------------------ | ------------------------------------------------ | ------------- | -------- | ---------------------------------------------- |
-| `enableAutoClose`  | `bool`                                           | `true`        | ✕        | Enables close on click outside of Dropdown     |
-| `fullWidthMode`    | [`DropdownFullwidthMode`][dropdownfullwidthmode] | `off`         | ✕        | Full-width mode                                |
-| `id`               | `string`                                         | `<random>`    | ✕        | Component id                                   |
-| `onAutoClose` .    | `(event: Event) => void`                         | —             | ✕        | Callback on close on click outside of Dropdown |
-| `placement`        | [Placement dictionary][dictionary-placement]     | `bottom-left` | ✕        | Alignment of the component                     |
-| `renderTrigger`    | `(render: DropdownRenderProps) => ReactNode`     | —             | ✕        | Properties for trigger render                  |
-| `UNSAFE_className` | `string`                                         | —             | ✕        | Wrapper custom classname                       |
-| `UNSAFE_style`     | `CSSProperties`                                  | —             | ✕        | Wrapper custom style                           |
+| Name               | Type                                             | Default       | Required | Description                                                                             |
+| ------------------ | ------------------------------------------------ | ------------- | -------- | --------------------------------------------------------------------------------------- |
+| `enableAutoClose`  | `bool`                                           | `true`        | ✕        | Enables close on click outside of Dropdown                                              |
+| `fullWidthMode`    | [`DropdownFullwidthMode`][dropdownfullwidthmode] | `off`         | ✕        | Full-width mode                                                                         |
+| `id`               | `string`                                         | `<random>`    | ✕        | Component id                                                                            |
+| `onAutoClose` .    | `(event: Event) => void`                         | —             | ✕        | Callback on close on click outside of Dropdown                                          |
+| `placement`        | [Placement dictionary][dictionary-placement]     | `bottom-left` | ✕        | Alignment of the component                                                              |
+| `renderTrigger`    | `(render: DropdownRenderProps) => ReactNode`     | —             | ✕        | [**DEPRECATED**][Deprecated] Will be removed in favor of new `DropdownModern` component |
```

Comment:
more like DropdownModernTrigger? and is it correct? From what I see it looks like the whole component is deprecated, so in the next major version, it will be removed.

Reply:
fixed